test/lib/find-scaffolds.js

```diff
@@ -36,45 +35,67 @@ const dependencies = {
     name: 'incorrectly-tagged',
     realPath: 'incorrectly-tagged realpath',
     keywords: ['cloverfield', 'something else']
+  },
+  packageWithoutName: {
+    // should skip this because it does not have a name
+    realPath: 'package-without-name realpath',
+    keywords: ['cloverfield', 'scaffold']
   }
 };
 
 
-const before = () => {
-  // Stubs
-  const load = spy();
-
-  stub(npm, 'load', load);
-  npm.commands = {
-    ls: spy()
+const mockNpm = () => {
+  const npm = {
+    load: spy(),
+    commands: {
+      ls: spy()
+    }
   };
 
-  return {load};
+  return {npm};
 };
 
 
-const after = () => {
-  npm.load.restore();
-};
+test('Find Scaffolds API', assert => {
+  const {npm} = mockNpm();
+
+  assert.ok(findGlobal(npm) instanceof Function, 'should be function');
+  assert.ok(findGlobal(npm)() instanceof Promise, 'should return Promise');
```

Comment:
These tests don't reveal the actual values in case of failure. Please read [5 Questions Every Unit Test Must Answer](https://medium.com/javascript-scene/what-every-unit-test-needs-f6cd34d9836d).